tribe-ext-calendar-widget-areas/index.php

```diff
@@ -1,17 +1,14 @@
 <?php
 /**
-* Plugin Name: The Events Calendar Extension: Calendar Widget Areas
-* Description: Adds widget areas (a.k.a. sidebars) that only display on The Events Calendar pages/views. Areas may be enabled or disabled at wp-admin > Events > Settings > Display tab > Advanced Template Settings section. Note that the WP Customizer only allows you to manage widget areas that apply to the page you're currently previewing; therefore, you will need to navigate to your Events page, for example, to edit the content of those widget areas via the Customizer's live preview.
-* Version: 1.0
-* Extension Class: Tribe__Extension__Calendar_Widget_Areas
-* Author: Modern Tribe, Inc.
-* Author URI: http://m.tri.be/1971
-* License: GPLv2 or later
-* License URI: https://www.gnu.org/licenses/gpl-2.0.html
-*/
-
-// Created 2015-10-05 for https://theeventscalendar.com/support/forums/topic/control-visibility-of-sidebar-widgets/#dl_post-1011743
-// Turned into an Extension by Cliff on 2017-01-11
+ * Plugin Name: The Events Calendar Extension: Calendar Widget Areas
+ * Description: Adds widget areas (a.k.a. sidebars) that only display on The Events Calendar pages/views. Areas may be enabled or disabled at wp-admin > Events > Settings > Display tab > Advanced Template Settings section. Note that the WP Customizer only allows you to manage widget areas that apply to the page you're currently previewing; therefore, you will need to navigate to your Events page, for example, to edit the content of those widget areas via the Customizer's live preview.
+ * Version: 1.0.0
```

Comment:
It looks like this should now be version 2.0.0 given the backwards incompatibility of the extension.